src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1797,6 +1798,38 @@ public TransformReason get(EntityTransformEvent event) {
 				return event.getTransformReason();
 			}
 		}, EventValues.TIME_NOW);
+		// EntityShootBowEvent
+		EventValues.registerEventValue(EntityShootBowEvent.class, ItemStack.class, new Getter<ItemStack, EntityShootBowEvent>() {
+			@Override
+			@Nullable
+			public ItemStack get(EntityShootBowEvent event) {
+				return event.getBow();
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(EntityShootBowEvent.class, Float.class, new Getter<Float, EntityShootBowEvent>() {
```

Comment:
it might be better to leave this just for ExprShootProperties. I don't think `event-float` is very useful